tests/src/test/java/alluxio/master/AlluxioMasterRestApiTest.java

```diff
@@ -57,178 +44,102 @@
 @PrepareForTest({AlluxioMaster.class, BlockMaster.class, Configuration.class, MasterContext.class,
     MetricRegistry.class, UnderFileSystem.class})
 public final class AlluxioMasterRestApiTest extends RestApiTest {
-  private static final String ALLUXIO_CONF_PREFIX = "alluxio";
-  private static final String NOT_ALLUXIO_CONF_PREFIX = "_alluxio_";
-  private AlluxioMaster mAlluxioMaster;
-  private BlockMaster mBlockMaster;
 
   @Before
   public void before() {
-    mAlluxioMaster = PowerMockito.spy(mResource.get().getMaster().getInternalMaster());
-    Whitebox.setInternalState(AlluxioMaster.class, "sAlluxioMaster", mAlluxioMaster);
-    mBlockMaster = PowerMockito.mock(BlockMaster.class);
-    Whitebox.setInternalState(mAlluxioMaster, "mBlockMaster", mBlockMaster);
     mHostname = mResource.get().getHostname();
     mPort = mResource.get().getMaster().getWebLocalPort();
     mServicePrefix = AlluxioMasterRestServiceHandler.SERVICE_PREFIX;
   }
 
   @Test
   public void getCapacityBytesTest() throws Exception {
-    Random random = new Random();
-    long capacityBytes = random.nextLong();
-    Mockito.doReturn(capacityBytes).when(mBlockMaster).getCapacityBytes();
-
+    long memorySize = Configuration.getBytes(Constants.WORKER_MEMORY_SIZE);
```

Comment:
is there a way to preserve the old behavior, mocking the service implementation?

the purpose of this test is to test the service API, not its implementation